packages/components/src/button/src/Button.css

```diff
@@ -60,7 +60,7 @@
 }
 
 /* TEXT */
-.o-ui-button .o-ui-button-text {
+.o-ui-button-text {
```

Comment:
How did you fix it? I don't see any :where() selector

Reply:
This is not yet fixed ;)